alf/algorithms/sac_algorithm.py

```diff
@@ -274,7 +290,10 @@ def __init__(self,
             )
 
         def _init_log_alpha():
-            return nn.Parameter(torch.tensor(float(initial_log_alpha)))
+            alpha = torch.tensor(float(initial_log_alpha))
+            if alpha_optimizer is None:
+                return alpha
+            return nn.Parameter(alpha)
```

Comment:
If `alpha_optimizer` is None, we can still create it as a parameter?

Reply:
> Could an overall optimizer accidentally include alpha parameter and optimize it?

I think the original intent of having `alpha_optimizer=None` is indeed to let the parent algorithm decides which optimizer to use for alpha, instead of not optimizing alpha at all.